content/v2.0/get-started.md

```diff
@@ -171,6 +171,8 @@ the [InfluxDB HTTP API](/v2.0/reference/api/).
 docker run --name influxdb -p 9999:9999 quay.io/influxdb/influxdb:2.0.0-beta
 ```
 
+Note: To run influxdb in detached mode use flag -d in the above command. 
```

Comment:
Quick edit:

```
_To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run/#detached-vs-foreground), include the `-d` flag in the `docker run` command._
```

Reply:
@VijetaH Once you apply this edit ☝️, we'll get this merged and deployed.

Reply:
@sanderson oops i resolved before applying edit. Its updated now.